source/assets/js/playground.ts

```diff
@@ -155,6 +155,12 @@ function setupPlayground(): void {
     }
   }
 
+  function goToSelection(selection: PlaygroundState['selection']): void {
```

Comment:
The type `PlaygroundState['selection']` is re-used often enough it's probably worth making its own named type.

Reply:
Done in https://github.com/sass/sass-site/pull/1179/commits/2d7ba875d1154d13cd07cdf950058ec55826d158